src/components/top-bar/styles.ts

```diff
@@ -3,6 +3,11 @@ import { isDesktop, isDesktopOrBelow, isTablet, isTabletOrBelow } from '../../st
 import { ElButtonGroup } from '../button-group'
 import { css } from '@linaria/core'
 
+export const ElTopBarLogo = styled.a`
+  padding-right: var(--spacing-2, 8px);
+  display: inline-flex;
```

Comment:
this `inline-flex` to make the logo vertically centered, `flex` also work though